src/EPR.Calculator.Frontend/Constants/ViewControlNames.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace EPR.Calculator.Frontend.Constants
+{
+    public static class ViewControlNames
+    {
+        public const string FileUpload = "#file-upload-1";
```

Comment:
Please remove # from the name

Reply:
@udaydefra  # is used in pointing cursor to error location when we click on error message. 
<li>
    <a href=@Model.DOMElementId>@Model.ErrorMessage</a>
</li>

Reply:
# need not be part of the constant. hence, asked you to remove